packages/extension/src/components/pages/SendPayment/ConfirmPayment/ConfirmPayment.tsx

```diff
@@ -79,7 +80,8 @@ export const ConfirmPayment: FC<ConfirmPaymentProps> = ({
         setTransaction(TransactionStatus.Success);
       })
       .catch((e) => {
-        setErrorRequestRejection(e.message);
+        const UIError = toUIError(e);
+        setErrorRequestRejection(UIError.message);
```

Comment:
No need to create a constant, it's used only one time:
```suggestion
        setErrorRequestRejection(toUIError(e).message);
```